mpp/include/mpp_chksum_int.h

```diff
@@ -89,9 +89,11 @@ function MPP_CHKSUM_INT_RMASK_( var, pelist, mask_val )
         write(unit=tmpstr3,fmt="(I1)") KIND(mask_val)
         errStr = errStr // tmpstr3 // ") mask_val="
         write(unit=tmpstr4,fmt=*) mask_val
-        errStr = errStr // trim(tmpstr4) // "has been called with these strange values. Check your KINDS, _FillValue, pack and mask_val. // &
-            & Hint: Try being explicit and using MPP_FILL_{INT,FLOAT,DOUBLE}. Continuing by using the default MPP_FILL_INT. // &
-            & THIS WILL BE FATAL IN THE FUTURE!"
+        errStr = errStr // trim(tmpstr4) // "has been called with these strange values. Check your KINDS, "// &
+              "_FillValue, pack and mask_val. "// &
+              "Hint: Try being explicit and using MPP_FILL_{INT,FLOAT,DOUBLE}. "// &
+              "Continuing by using the default MPP_FILL_INT. " // &
+              "THIS WILL BE FATAL IN THE FUTURE!"
```

Comment:
Since this is within a string, probably should probably use an & right in front of the continued line as well.